src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -59,6 +59,7 @@ end
 if peripheral.find( "printer" ) then
     table.insert( tMenuItems, "Print" )
 end
+table.insert( tMenuItems, "Jump" )
```

Comment:
Ok then. Would you say remove it then or just make a setting for it that is by default false? Like `edit.advanced_functionality` ?